conf/app.routes

```diff
@@ -12,4 +12,8 @@ GET       /you-are-eligible                           @uk.gov.hmrc.helptosavestr
 
 GET       /account-already-exists                     @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.accountAlreadyExists()
 
-GET       /terms-and-conditions                       @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.getTermsAndConditionsPage()
\ No newline at end of file
+GET       /terms-and-conditions                       @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.getTermsAndConditionsPage()
+
+GET       /details-confirmed                          @uk.gov.hmrc.helptosavestridefrontend.controllers.StrideController.handleDetailsConfirmed()
```

Comment:
this should be a post if this is the action of the button rather than a page - that way we can make the endpoint more secure as a csrf token will be required